proto/regen/data/v1/tx.proto

```diff
@@ -10,18 +10,18 @@ option go_package = "github.com/regen-network/regen-ledger/x/data";
 
 // Msg is the regen.data.v1alpha1 Msg service
 service Msg {
-  // AnchorData "anchors" a piece of data to the blockchain based on its secure
+  // Anchor "anchors" a piece of data to the blockchain based on its secure
```

Comment:
```suggestion
  // Anchor a piece of data to the blockchain based on its secure
```

Reply:
`Anchor` is the name of the method here, not a verb.